tests/TestCase/Installer/PluginInstallerTest.php

```diff
@@ -19,7 +19,17 @@ class PluginInstallerTest extends TestCase
      *
      * @var string
      */
-    protected $testDirs = ['', 'vendor', 'plugins', 'plugins/Foo', 'plugins/Fee', 'plugins/Foe', 'plugins/Fum'];
+    protected $testDirs = [
+        '',
+        'vendor',
+        'plugins',
+        'plugins/Foo',
+        'plugins/Fee',
+        'plugins/Foe',
+        'plugins/Fum',
+        'app_plugins',
+        'app_plugins/Bar',
```

Comment:
Do these new plugins need any files/directories added?

Reply:
Nope, the installer just reads the directories under a plugin directory and adds them to plugin list.

Reply:
Ok.